input/intro-notes/StructureDefinition-au-practitioner-intro.md

```diff
@@ -5,3 +5,13 @@
 - A tertiary qualification or professional membership (non-Ahpra-sourced data) is represented by `Practitioner.qualification` 
   - If none of the codes from the preferred value set are suitable then at least text should be sent in `Practitioner.qualification.code`
 - See each Identifier profile page for guidance related to that identifier type.
+- This profile supports the practitioner's gender identity aligned to [Australian Bureau of Statics Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) using the [Individual Gender Identity extension](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) and [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set.
```

Comment:
Remove the references to specific (and incorrect) examples.

Reply:
Done